recipes/pysurvival/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pysurvival" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57cb567323fa314ceaf17fe39745b007855606cb1304a4124a78334ff9d03707
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py>38 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - pandas
+    - pip
+    - progressbar
+    - pyarrow
+    - python
+    - scikit-learn
+    - scipy
+    - pytorch >=1.7.0
+    - {{ pin_compatible('numpy') }}
+    - {{ compiler('c') }}
```

Comment:
Are you able to confirm that you do need compilers at runtime too? This is generally odd for packages but some do require the C and C++ compilers at runtime.

Reply:
I think so, since it is building shared libraries.

Reply:
the build happens once, and then conda packages include the shared libraries. At runtime, you very likely do not need the compilers.

Compilers are only needed if the application will compile new C code at runtime. This is quite rare but does exist (numba for example is one such package)

Reply:
Okay! I have removed them in the new version.